src/common/helpers/responseHelper.ts

```diff
@@ -0,0 +1,70 @@
+import { Response } from "express";
+
+enum Status {
+    Error400 = 400,
+    Error401 = 401,
+    Error404 = 404,
+    Error422 = 422,
+    Error500 = 500,
+    Success200 = 200,
+}
+
+export function BadRequest(res: Response, reasonString: string): Response {
```

Comment:
To reduce complexity, I recommend having a single response helper (called `ResponseBuilder` for example) that takes in the `Status` enum as a parameter, then uses a switch statement to build the appropriate response.